paywall/src/constants.js

```diff
@@ -44,8 +44,8 @@ export const ACCOUNT_REGEXP = new RegExp(accountRegex)
 
 // helpers for the LOCK_PATH_NAME_REGEXP
 const prefix = '[a-z0-9]+'
-const urlEncodedRedirectUrl = '[^#?]+'
-const userAccount = accountRegex
+const urlEncodedUrl = '[^#?]+'
+const urlEncodedRedirectUrl = urlEncodedUrl
```

Comment:
I think you do not need that? The constant name should describe what it is, not what it can be used for?

Reply:
👍